CordovaLib/Classes/Private/Plugins/CDVWebViewEngine/CDVWebViewUIDelegate.m

```diff
@@ -160,4 +159,29 @@ - (void)webViewDidClose:(WKWebView*)webView
 }
 
 
++ (UIViewController *)topViewController {
```

Comment:
Is there a reason for this to be a static method on `CDVWebViewUIDelegate` rather than an instance method used with `[self topViewController]`? It looks like all uses of it are within the `CDVWebViewUIDelegate` class